docs/for-developers/technical-overview.md

```diff
@@ -3,71 +3,200 @@ id: technical-overview
 
-<i>DaiDripsHub is part of the <a href="https://github.com/radicle-dev/radicle-drips-hub/blob/master/src/DaiDripsHub.sol">Radicle-Drips-Hub</a> repository.</i>
+### Attributes of Drips
 
-The <a href="https://github.com/radicle-dev/radicle-drips-hub/">DaiDripsHub</a> smart contract is the "lowest-level" component in the Drips architecture. DaiDripsHub provides extremely general, standalone, functionality for allowing any Ethereum address to schedule one-time or streaming payments to
-any other Ethereum address. In this case, "streaming" payments means payments that are made gradually over time -
-something that is not possible using the standard Ethereum transactions.
+A drip is defined by the following attributes
+- **sender** - A Drips user, typically an Ethereum address 
+- **receiver** - A Drips user, typically an Ethereum address
+- **amount per second** - uint
+- **start** - unix timestamp
+- **end** - unix timestamp
 
-An example of a streaming payment would be scheduling a recurring monthly payment from one user to another. However many other types of streaming payments are also possible and for that reason we believe that DaiDripsHub can be a poweful building block for any web3 app looking to implement transfers of value that happen gradually over time.
+### Scheduled Drips
 
-### The RadicleRegistry Smart Contract
+When a Drip is configured, the sender can decide whether funds will start flowing immediately (i.e. in the same block that the transaction is processed)
+or whether the flow of funds will be **scheduled** to start at some future time. Similarly, the duration of the Drip can be set explicitly, or it can be configured
+to stream funds until the user either stops the Drip, or funds run out. We refer to Drips with an explicitly defined start time or duration as "scheduled".
 
-<i>RadicleRegistry is part of the <a href="https://github.com/radicle-dev/radicle-drips/">Radicle-Drips</a> repository.</i>
+### What are Splits?
 
-The <a href="https://github.com/radicle-dev/radicle-drips/blob/master/src/registry.sol">RadicleRegistry</a> smart
-contract builds on DAIDripsHub to allow users to create and deploy DripsToken ERC-721 Smart Contracts (described below). The main purpose of the registry is to ensure that all Communities share the same underlying contract format and utilize the same DaiDripsHub contract, so that they can all Drip to one another interchangably.
+The Drips protocol also includes built-in features for sharing of funds, or "Splitting". When a user creates a Split, a percentage of all funds sent to them though the Drips protocol will be automatically split to another user of their choice. More precisely, an Ethereum address (the receiver) can define a set of split addresses with percentages:
```

Comment:
A splits receiver "is a" user ID. Should we remove the word "address" to avoid confusion with the `AddressApp`?

I guess this is a broader question for the technical overview section. 
Do we want to describe the `DripsHub` _protocol_? Or Drips "in general"? 
Point is, somehow to avoid mixing Drips/DripsHub/AddressApp when describing how it works.

Reply:
Yes totally. Again, this is tough. My thought is to do a whole separate page on hub identities, AddressApp and any other standard apps we define where we would explain things at that level of nuance... while trying to keep things simpler and mostly discuss users in the context of Ethereum addresses in this main technical overview page (which will already have quite a few new/challenging ideas for many readers).

Reply:
Once we have this new page, I think it could make sense to add little blurbs/links in various sections (including this one) like "See the Users and Hub Apps section for full details on how users work in Drips".